processor/src/operations/crypto_ops.rs

```diff
@@ -491,6 +491,55 @@ mod tests {
         assert!(process.advice.has_merkle_root(expected_root));
     }
 
+    #[test]
+    fn op_mrupdate_invalid_path_length() {
```

Comment:
This is under the old processor; can you move it under the new processor instead (`processor/src/procesor/operations/crypto_ops.rs`)?